src/components/Timeline/TimelineIcon.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+
+import { makeStyles } from '@material-ui/core/styles'
+import TimelineItem from '@material-ui/lab/TimelineItem'
+import TimelineSeparator from '@material-ui/lab/TimelineSeparator'
+import TimelineConnector from '@material-ui/lab/TimelineConnector'
+import TimelineContent from '@material-ui/lab/TimelineContent'
+import TimelineDot from '@material-ui/lab/TimelineDot'
```

Comment:
Haven't we exported everything from cozy-ui?

Reply:
Non rien dans cozy-ui sur le sujet à l'heure actuelle, une fois que ce sera sec dans cette app on devrait tout migrer dans cozy-ui (il reste le clic sur les lignes qui n'existe pas encore)

Reply:
Oh yep I see. `/lab/` ... 🤒